lms/templates/staff_problem_info.html

```diff
@@ -51,9 +51,9 @@
     </form>
 
   </div>
-</section>
+</div>
 
-<section aria-hidden="true" class="modal staff-modal" id="${element_id}_debug" style="width:80%; left:20%; height:80%; overflow:auto; display:none" >
+<div aria-hidden="true" class="modal staff-modal" id="${element_id}_debug" style="width:80%; left:20%; height:80%; overflow:auto; display:none" >
```

Comment:
@cahrens These are changed to role="dialog" and a11y team already reviewed them, i got those changes in recent rebase.